lib/modules/manager/nuget/update.ts

```diff
@@ -29,23 +27,19 @@ export function bumpPackageVersion(
   }
 
   try {
-    const project = new XmlDocument(content);
-    const versionNode = project.descendantWithPath('PropertyGroup.Version')!;
-    const startTagPosition = versionNode.startTagPosition;
-    const versionPosition = content.indexOf(versionNode.val, startTagPosition);
-
     const newProjVersion = semver.inc(currentValue, bumpVersion as ReleaseType);
     if (!newProjVersion) {
       throw new Error('semver inc failed');
     }
 
     logger.debug(`newProjVersion: ${newProjVersion}`);
-    bumpedContent = replaceAt(
```

Comment:
no, `replaceAt` needs to be fixed

Reply:
Added a check in nuget before even calling `bumpVersion()`. Also, fixed replaceAt, but not checked for performance impact.